apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,26 @@
+import { Navigate } from "~/libs/components/components.js";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+type Properties = {
+	component: React.ReactNode;
+	redirectTo?: ValueOf<typeof AppRoute>;
+};
+
+const ProtectedRoute: React.FC<Properties> = ({
+	component,
+	redirectTo = AppRoute.SIGN_IN,
+}: Properties) => {
```

Comment:
```suggestion
}) => {
```
`React.FC<Properties>` already handles this

Reply:
No, it does not. There are lint errors with missing react/props and I looked through other components to find this solution with duplicating Properties

Reply:
You're right, sorry for misunderstanding

Reply:
Right, this is a part of `eslint-react` config